### PR TITLE
Blademaster actions to Duelist 2

### PR DESCRIPTION
**Pull request by Eliemer**
# Description

Added all the actions available from `Duelist II: Blademaster`

## Issue Number
N/A

## Type of change

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: lib/talents.json [hunk 1]
```diff
@@ -371,6 +371,30 @@
             "detail": "1/round, when you hit with a Main Melee weapon, gain 1 Blademaster Die."
           }
         ],
+        "actions": [
+          {
+            "name": "Parry",
+            "activation": "Reaction",
+            "trigger": "You are hit by a melee attack",
+            "detail": "You gain Resistance to all damage, heat, and burn dealt by the attack"
```
Comment (msprijatelj) [inline lib/talents.json]: Minor nitpick: End the sentences with a `.` for consistency with the other actions.

File: lib/talents.json [hunk 2]
```diff
@@ -371,6 +371,30 @@
...
+          },
+          {
+            "name": "Deflect",
+            "activation": "Reaction",
+            "trigger": "You are hit by a ranged attack",
+            "detail": "You may roll any number of Blademaster Dice, expending them: if you roll a 5+ on any of these dice, you gain Resistance to all damage, heat, and burn dealt by the attak."
```
Comment (msprijatelj) [inline lib/talents.json]: Typo; change `attak` to `attack`.

File: lib/talents.json [hunk 3]
```diff
@@ -371,6 +371,30 @@
...
+          },
+          {
+            "name": "Feint",
+            "activation": "Free",
+            "detail": "Choose an adjacent character: when moving, you ignore engagement and don't provoke reactions from your target until the start of your next turn."
```
Comment (msprijatelj) [inline lib/talents.json]: Another nitpick: The `'` character in `don't` is originally a `’` character in the main text (so `don’t`). Incredibly minor, but for consistency I recommend just copy/pasting the curly apostrophe from the main description to these details (Feint and Trip).

**Review by msprijatelj — COMMENTED** [review]
Thanks for adding these! I think it may be good to mention how many blademaster dice you need to roll for each option. For example, for Parry:
```
Expend 1 Blademaster Die. You gain Resistance to all damage, heat, and burn dealt by the attack.
```
Otherwise, there's a few typos here and there, but otherwise good!

**Review by msprijatelj — APPROVED** [review]

**Comment by Eliemer** [comment]
image
It looks pretty nice!

**Pull request merged**